ui/component/livestreamComments/view.jsx

```diff
@@ -64,6 +77,16 @@ export default function LivestreamComments(props: Props) {
 
   const pinnedComment = pinnedComments.length > 0 ? pinnedComments[0] : null;
 
+  const shouldRefreshTimestamp =
+    commentsByChronologicalOrder &&
+    commentsByChronologicalOrder.some((comment) => {
+      const { timestamp } = comment;
+      const timePosted = timestamp * 1000;
+      const { suffix } = getTimeAgoData(timePosted);
```

Comment:
We can skip the heavy `getTimeAgoData` since we just want to know if the timestamp is in the "minutes" range:
- Put a `const now = Date.time()` outside the loop (or `new Date()`? not sure which one matches `timestamp`)
- In the loop, compare/subtract with `timestamp`, and return true if less than 1 hour.